api/src/main/java/org/pmiops/workbench/utils/WorkspaceMapper.java

```diff
@@ -59,6 +59,9 @@ void mergeResearchPurposeIntoWorkspace(
       @MappingTarget DbWorkspace dbWorkspace, ResearchPurpose researchPurpose);
 
   @Mapping(target = "timeReviewed", ignore = true)
+  @Mapping(target = "disseminateResearchFindingList", ignore =  true)
```

Comment:
Why are we ignoring these?

Reply:
+1

Also, please add/check test cases in `WorkspaceMapperTest`.